### DISCUSSION TITLE
Allow combining roles

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5604] [Feature Requests]
When creating a new role, there should be an option to create a new role based on multiple existing roles

**Comment by maxpatiiuk** [comment]
Don't remember who was this requested by. Also, what are their use cases?

Is simply concating the list of policies from both roles going to work, or do they need any changes to sub roles to update the super role?

**Comment by grantfitzsimmons** [comment]
Going to close as not planned unless this is discussed again. I do not recall who requested this ability.

**Comment by grantfitzsimmons** [comment]
From Corinna P at CSIRO:
> I believe we requested this back when we had a long discussion about the new permissions system. We basically don't use the fine tuning abilities of the system as it's a major overhead for us to break down permissions into 'functions' and then assign a large number of roles per user (and per collection if there's more than one in an instance). 
>
> Instead, we pretty much replicated the 'old' system where we have one role per permission lot.
>
>I think the combined role functionality would have a number of benefits and if it became available we'd switch over to that as soon as possible. I understand that it isn't a high priority item, but it would be good to keep the ticket open in case you have some time to address that in the future.
